src/sentry/api/event_search.py

```diff
@@ -948,47 +992,42 @@ def is_function(field):
 
 def get_function_alias(function_name, columns):
     columns = "_".join(columns).replace(".", "_")
-    return ("%s_%s" % (function_name, columns)).rstrip("_")
+    return u"{}_{}".format(function_name, columns).rstrip("_")
 
 
-def resolve_function(field, match=None):
+def resolve_function(field, match=None, params=None):
     if not match:
         match = FUNCTION_PATTERN.search(field)
         if not match or match.group("function") not in FUNCTIONS:
-            raise InvalidSearchQuery("%s is not a valid function" % field)
+            raise InvalidSearchQuery(u"{} is not a valid function".format(field))
 
     function = FUNCTIONS[match.group("function")]
-    columns = [c.strip() for c in match.group("columns").split(",")]
+    columns = [c.strip() for c in match.group("columns").split(",") if len(c.strip()) > 0]
+
+    # Some functions can optionally take no parameters (rpm(), rps()). In that case use the
+    # passed in params to create a default argument if necessary.
+    if len(columns) == 0 and len(function["args"]) == 1:
+        try:
+            default = function["args"][0].has_default(params)
+        except InvalidFunctionArgument as e:
+            raise InvalidSearchQuery(u"{}: invalid arguments: {}".format(field, e))
+
+        if default:
+            # Hacky, but we expect column arguments to be strings so easiest to convert it back
+            columns = [six.binary_type(default)]
```

Comment:
This will give you binary strings in py3. You might want `text_type()` to ensure you don't have to deal with encoding problems later.